gulpfile.ts

```diff
@@ -19,13 +19,13 @@ import * as gulp from 'gulp';
 import * as nodemon from 'gulp-nodemon';
 import * as runSequence from 'run-sequence';
 import tslint from 'gulp-tslint';
-import * as shell from 'gulp-shell';
 import * as tsc from 'gulp-typescript';
 
 const webpack = require('webpack-stream');
+const { exec } = require('child_process');
```

Comment:
Can we use `import * as childProcess from 'child_process';`

Reply:
OK, I'll change this.